### PR TITLE
[Fix] Game status not updating on uninstall

### PR DESCRIPTION
**Pull request by flavioislima**
The issue was due to the dependencies inside `useEffect` on the Game Page.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/components/UI/CachedImage/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const CachedImage = (props: Props) => {
   let src = useFallback ? props.fallback : props.src
   src = useCache ? `imagecache://${src}` : src
 
-  return <img {...props} src={src} onError={onError} />
+  return <img loading="lazy" {...props} src={src} onError={onError} />
```
Comment (flavioislima) [inline src/frontend/components/UI/CachedImage/index.tsx:30]: Not related with this PR but adding here since it is just one line of code.

**Review by Nocccer — APPROVED** [review]

**Comment by arielj** [comment]
hey @flavioislima can you explain the issue this is fixing or steps to reproduce it? I can't reproduce the problem in the main branch if I understand it correctly, when I uninstall a game it's updating the GamePage with no issues

**Comment by flavioislima** [comment]
> hey @flavioislima can you explain the issue this is fixing or steps to reproduce it? I can't reproduce the problem in the main branch if I understand it correctly, when I uninstall a game it's updating the GamePage with no issues

Sometimes it updates and sometimes not.
I added some examples in the development channel in discord.
If we use only Gog and epic in the array of dependencies it trigger up to 5 re-renders and this makes the value not update sometimes.
Using epic.library and gog.library makes it re-render only once and I could not reproduce this error anymore.

**Review by arielj — APPROVED** [review]
Looks good, I can't reproduce the original issue but the code changes look fine to me.

**Pull request merged**